.github/workflows/tests.yml

```diff
@@ -0,0 +1,49 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '07 01 * * *'
+
+jobs:
+  unit-tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9, pypy3]
```

Comment:
Are you aware that you can pass that value dynamically? 

See example: https://github.com/apache/airflow/blob/ecc3a4df0da67f258c3ad04733d6e561d8266c93/.github/workflows/ci.yml#L485

In this case, you can keep information about the supported Python version in one place, eg in tox.ini.

Reply:
I'd have to write and maintain a script to extract that info from tox. No thanks.